addon/pods/components/rui-validatable-input/template.hbs

```diff
@@ -11,9 +11,8 @@ name=valuePath}}
   </p>
 {{/if}}
 
-{{#if showMessage}}
+{{#if isInvalid}}
```

Comment:
I feel like there should be an option to hide the messages, but I don't know if that is part of the scope of this.